core/trino-main/src/main/java/io/trino/operator/project/PageProcessor.java

```diff
@@ -95,11 +95,13 @@ public PageProcessor(Optional<PageFilter> filter, List<? extends PageProjection>
         this(filter, projections, OptionalInt.of(1));
     }
 
+    @VisibleForTesting
     public Iterator<Optional<Page>> process(ConnectorSession session, DriverYieldSignal yieldSignal, LocalMemoryContext memoryContext, Page page)
     {
         return process(session, yieldSignal, memoryContext, page, false);
     }
 
+    @VisibleForTesting
     public Iterator<Optional<Page>> process(ConnectorSession session, DriverYieldSignal yieldSignal, LocalMemoryContext memoryContext, Page page, boolean avoidPageMaterialization)
```

Comment:
This one can be reduced to package-level visibility

Reply:
done in another commit